thirdeye/thirdeye-frontend/app/pods/components/range-pill-selectors/component.js

```diff
@@ -50,46 +50,35 @@ export default Component.extend({
   /**
    * A set of arbitrary time ranges to help user with quick selection in date-time-picker
    */
+  // eslint-disable-next-line ember/avoid-leaking-state-in-ember-objects
   predefinedRanges: {
```

Comment:
You can define this object as a constant outside the component and then reference it here to avoid having to bypass eslint.